objects/rct2ww/ride/rct2.ww.minecart.json

```
@@ -78,6 +79,7 @@
             "sv-SE": "Berg- och dalbanevagnar av trä med stoppade säten och säkerhetsbälten",
             "zh-CN": "有装填物的座位及护膝吧装置的木制云霄飞车",
             "cs-CZ": "Dřevěná horská dráha s vozy s polstrovanými sedadly a opěrkou v klíně",
+            "ja-JP": "膝の安全バーが付いた、パッド入りの座席を持つ木製のコースターですが、鉱山カートのような外観があります"
```

Missing trailing comma:
```suggestion
            "ja-JP": "膝の安全バーが付いた、パッド入りの座席を持つ木製のコースターですが、鉱山カートのような外観があります",
```